website/docs/r/resource_instance.html.markdown

```diff
@@ -35,6 +35,33 @@ resource "ibm_resource_instance" "resource_instance" {
 }
 ```
 
+The following is an example to create a service instance of IBM Cloud Hyper Protect DBaaS for PostgreSQL. For detailed argument reference, see the tables in the [Hyper Protect DBaaS for PostgreSQL documentation](/docs/hyper-protect-dbaas-for-postgresql?topic=hyper-protect-dbaas-for-postgresql-create-service#cli-create-service) (or the [Hyper Protect DBaaS for MongoDB documentation](/docs/hyper-protect-dbaas-for-mongodb?topic=hyper-protect-dbaas-for-monbgodb-create-service#cli-create-service) if you want to create MongoDB service instances).
+
+```
+data "ibm_resource_group" "group" {
+  name = "default"
+}
+resource "ibm_resource_instance" "myhpdbcluster" {
+  name = "0001-postgresql"
+  service = "hyperp-dbaas-postgresql"
+  plan = "postgresql-free"
+  location = "us-south"
+  resource_group_id = data.ibm_resource_group.group.id
+  timeouts {
+    create = "15m"
+    update = "15m"
+    delete = "15m"
+  }
+  parameters = {
```

Comment:
parameters shouldnot have = it doesn't work from terraform v12 and higher version.....Terraform11 use to support this..

parameters {
------
}